include/LightGBM/c_api.h

```diff
@@ -759,7 +759,11 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterGetNumClasses(BoosterHandle handle,
 /*!
  * \brief Update the model for one iteration.
  * \param handle Handle of booster
- * \param[out] is_finished 1 means the update was successfully finished (cannot split any more), 0 indicates failure
+ * \param[out] is_finished 1 means the tree(s) produced by this iteration did not have any splits.
```

Comment:
I don't think that it worth introducing a breaking change for renaming this param. I believe that changing the description is enough.

Reply:
I agree with you, that place should be left unchanged because it has a different meaning there.... for now.

That will change in #5699